modules/test/playwright/pages/headless-discovery-web/apiExplorerPage.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page, expect} from '@playwright/test';
+
+import {liferayConfig} from '../../liferay.config';
+
+export class ApiExplorerPage {
+	readonly page: Page;
+
+	constructor(page: Page) {
+		this.page = page;
+	}
+
+	async endpointHasSpecificParameters(
```

Comment:
Rename to `endpointHasParameters`. The `specific` word adds no meaning. Simpler is better.